airflow-core/src/airflow/ui/public/i18n/locales/de/common.json

```diff
@@ -89,6 +89,7 @@
     "back": "Zurück",
     "defaultMessage": "Ein unerwarteter Fehler ist aufgetreten",
     "home": "Start",
+    "invalidUrl": "Seite nicht gefunden. Bitte überprüfen Sie die URL und versuchen Sie es erneut.",
```

Comment:
I'm curious, what the difference between `invalidUrl` and `notFound`? (rather unrelated to the German translation itself)

Reply:
Thanks! :)